synfig-studio/src/synfigapp/actions/waypointsetsmart.cpp

```diff
@@ -162,14 +182,17 @@ Action::WaypointSetSmart::set_param(const synfig::String& name, const Action::Pa
 bool
 Action::WaypointSetSmart::is_ready()const
 {
-	if(!value_node)
-		synfig::error("Missing value_node");
+	//we either need a valuedesc for the inital waypoint added
+	//or a valuenode to add more waypoints to the parameter
+	if(!value_desc && !value_node){
+		synfig::error("Missing both value_desc and value_node");
+		return false;
+	}
 
-	if(waypoint.get_time()==(Time::begin()-1))
+	if(waypoint.get_time()==(Time::begin()-1)){
 		synfig::error("Missing waypoint");
```

Comment:
Surround error message with `_()` for translation purposes.

Reply:
Done